roles/etcd/defaults/main.yaml

```diff
@@ -89,7 +91,7 @@ etcd_listen_client_urls: "{{ etcd_url_scheme }}://{{ etcd_ip }}:{{ etcd_client_p
 #etcd_peer: 127.0.0.1
 etcdctlv2: "{{ r_etcd_common_etcdctl_command }} --cert-file {{ etcd_peer_cert_file }} --key-file {{ etcd_peer_key_file }} --ca-file {{ etcd_peer_ca_file }} -C https://{{ etcd_peer }}:{{ etcd_client_port }}"
 
-etcd_service: etcd
+etcd_service: "{{ (r_etcd_common_etcd_runtime == 'docker') | ternary('etcd_container', 'etcd') }}"
```

Comment:
I don't think we need this.  We're brute-force removing the services in favor of pods, except for stand-alone rpm etcd.

Reply:
Yep, this currently breaks, so I'll remove etcd commits